spec/features/phone/add_phone_spec.rb

```diff
@@ -97,12 +103,25 @@
     expect(page).to have_content(t('forms.two_factor.code'))
   end
 
+  scenario 'Displays an error message when max phone numbers are reached' do
+    allow(IdentityConfig.store).to receive(:max_phone_numbers_per_account).and_return(1)
+    user = create(:user, :signed_up)
+    sign_in_and_2fa_user(user)
+    within('.sidenav') do
+      click_on t('account.navigation.add_phone_number')
+    end
+    expect(page).to have_css('#phones.usa-alert.usa-alert--error')
+    expect(page).to have_content(t('users.phones.error_message'))
```

Comment:
Minor: We could ensure that the content exists within the element we're expecting by using the `text` keyword argument of `have_css`:

```suggestion
    expect(page).to have_css(
      '#phones.usa-alert.usa-alert--error',
      text: t('users.phones.error_message'),
    )
```

https://rubydoc.info/github/jnicklas/capybara/Capybara/Node/Matchers#has_css%3F-instance_method

Reply:
That's a good catch 👍🏼